recipes/poppler-data/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class PopplerDataConan(ConanFile):
+    name = "poppler-data"
+    description = "encoding files for use with poppler, enable CJK and Cyrrilic"
+    homepage = "https://poppler.freedesktop.org/"
+    topics = "conan", "poppler", "pdf", "rendering"
+    license = "BSD-3-Clause", "GPL-2.0-or-later", "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
```

Comment:
Maybe add settings to avoid warnings?

```
[HOOK - conan-center.py] pre_export(): WARN: [HEADER_ONLY, NO COPY SOURCE (KB-H005)] This recipe is a header only library as it does not declare 'settings'. Please include 'no_copy_source' to avoid unnecessary copy steps
poppler-data/0.4.9: WARN: compiler setting should be defined.
poppler-data/0.4.9: WARN: build_type setting should be defined.
poppler-data/0.4.9: WARN: compiler setting should be defined.
poppler-data/0.4.9: WARN: compiler setting should be defined.
poppler-data/0.4.9 (test package): WARN: This conanfile has no build step
```